plugins/board-resources/src/index.ts

```diff
@@ -52,6 +54,12 @@ async function showCardLabelsPopup (object: Card): Promise<void> {
   showPopup(CardLabelsPopup, { object })
 }
 
+async function showEditMembersPopup(object: Card, client: TxOperations): Promise<void> {
+  const onClose = () => closePopup()
```

Comment:
Why you need this trick?

Reply:
I can't pass `closePopup` directly in `openPopup` because it's not initialized yet when i call `openPopup`, but i can pass initialized `onClose` and access `closePopup` from `onClose`'s context when it will be called